src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -207,6 +208,164 @@ void wrapUp(GitHub root) {
         }
     }
 
+    /**
+     * Represents a repository object specifically for {@link Installation} and {@link InstallationRepositories}.
+     */
+    @SuppressFBWarnings(value = { "UWF_UNWRITTEN_FIELD" }, justification = "JSON API")
+    public static class InstallationRepository {
+        private long id;
+        private String nodeId;
+        private String name;
+        private String fullName;
+        @JsonProperty("private")
+        private boolean _private;
+
+        /**
+         * Gets id
+         *
+         * @return the id
+         */
+        public long getId() {
+            return id;
+        }
+
+        /**
+         * Gets node id
+         *
+         * @return the node id
+         */
+        public String getNodeId() {
+            return nodeId;
+        }
+
+        /**
+         * Gets name
+         *
+         * @return the name
+         */
+        public String getName() {
+            return name;
+        }
+
+        /**
+         * Gets full name
+         *
+         * @return the full name
+         */
+        public String getFullName() {
+            return fullName;
+        }
+
+        /**
+         * Is private boolean.
+         *
+         * @return the boolean
+         */
+        public boolean isPrivate() {
+            return _private;
+        }
+    }
+
+    /**
+     * An installation has been installed, uninstalled, or its permissions have been changed.
+     *
+     * @see <a href="https://developer.github.com/v3/activity/events/types/#installationevent">authoritative source</a>
+     */
+    @SuppressFBWarnings(value = { "UWF_UNWRITTEN_FIELD" }, justification = "JSON API")
+    public static class Installation extends GHEventPayload {
+        private String action;
+        private GHAppInstallation installation;
+        private List<InstallationRepository> repositories;
+
+        /**
+         * Gets action
+         *
+         * @return the action
+         */
+        public String getAction() {
+            return action;
+        }
+
+        /**
+         * Gets installation
+         *
+         * @return the installation
+         */
+        public GHAppInstallation getInstallation() {
+            return installation;
+        }
+
+        /**
+         * Gets repositories
+         *
+         * @return the repositories
+         */
+        public List<InstallationRepository> getRepositories() {
+            return repositories;
+        };
+    }
```

Comment:
@XiongKezhi  You'll want to add an override for `wrapUp(root)` to bind `GHAppInstallation`